### DISCUSSION TITLE
Security advisories in github

### DISCUSSION THREAD

**Discussion by sheldonhull** [renovatebot/renovate#19330] [Suggest an Idea]
GitHub released a new feature that allows private security advisor reached views, rather than opening issues. Since GitHub doesn't have the concept of a private issue in a public repo, this is a great way to maintain a little more security.

It would be great if mend and renovate could create those instead of issues in github so you can track resolution internally.

https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability

**Comment by rarkins** [comment]
It's been enabled for about a week already, and I just confirmed it's visible:

<img width="1379" alt="image" src="https://user-images.githubusercontent.com/6311784/206764242-acd67002-809d-4052-945f-2e5f09caf87f.png">

For now while it's still in beta we'll leave our disclosure email address as the primary way, but of course people can submit privately via GitHub

**Reply by sheldonhull** [reply]
Just as a follow-up, this was more of a request that the issue alerting with renovate could leverage this so the vulnerability issue creation doesn't show publicly. Right now, I don't leverage any issue creation because I don't want that showing up for everyone.
